lib/Mojo/WebSocketProxy/Backend/ConsumerGroups.pm

```diff
@@ -201,7 +201,9 @@ sub call_rpc {
             }
             catch {
                 my $error = $@;
-                $rpc_failure_cb->($c, $result, $req_storage) if $rpc_failure_cb;
+                $rpc_failure_cb->($c, $result, $req_storage, {
+                        type => 'WrongResponse',
+                    }) if $rpc_failure_cb;
```

Comment:
we should have replicate same behavior like we have on JSONRPC
https://github.com/binary-com/perl-Mojo-WebSocketProxy/blob/master/lib/Mojo/WebSocketProxy/Backend/JSONRPC.pm#L126

Reply:
The only difference between them is error object params since other params (message, code) are extra params because they are useful only in case of `CallError`s unless we would like to keep behavior the same due to consistency.

Reply:
@mehdi-fs better to pass `code` and `message` in sake of consistency.